CHANGELOG.md

```diff
@@ -1,5 +1,9 @@
 # Changelog
 
+### 2.6.5-a
```

Comment:
Could we have a more descriptive tag than `-a` please? 🙏

Reply:
We purposely wanted to keep it vague -- hoping no one else pulls this

Reply:
Why not just call it `2.7.0-beta-1` or something then? 😝